.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @dylanj1752 @felddy @INLGuy @itsmostafa @jsf9k @mcdonnnj @xlyk @izzy64
+* @dav3r @dylanj1752 @felddy @INLGuy @itsmostafa @izzy64 @jsf9k @mcdonnnj @xlyk
```

Comment:
@felddy @jsf9k @mcdonnnj - Do we want to remove our team from the list of owners here as long as they are making updates?

Reply:
Seems like no one on my team seems to mind being code owners here, so feel free to leave us in for now.